homeassistant/components/mikrotik/device_tracker.py

```diff
@@ -149,29 +150,23 @@ def update_device_tracker(self):
                 if attr in device and device[attr] is not None:
                     attrs[slugify(attr)] = device[attr]
 
-            attrs["scanner_type"] = self.method
-            attrs["scanner_host"] = self.host
-            attrs["scanner_hostname"] = self.hostname
             self.device_tracker[mac] = attrs
 
-    def do_arp_ping(self, mac, interface):
+    def do_arp_ping(self, ip_address, interface):
         """Attempt to arp ping MAC address via interface."""
         params = {
             "arp-ping": "yes",
             "interval": "100ms",
             "count": 3,
             "interface": interface,
-            "address": mac,
+            "address": ip_address,
         }
         cmd = "/ping"
         data = self.api.command(cmd, params)
         if data is not None:
             status = 0
             for result in data:
                 if "status" in result:
-                    _LOGGER.debug(
-                        "Mikrotik %s arp_ping error: %s", self.host, result["status"]
-                    )
```

Comment:
Are the debug details no longer useful?

Reply:
We can add that back in. Any time an arp_ping fails it prints this debug.